docs/prob/api/record-not-found.md

```diff
@@ -29,7 +29,7 @@ looking for cannot be found. Such as if you've made a typo in the URL or some
 bug in the UI.
 
 This is what is commonly known as the "404 error", as [the status code 404
-stands for "Not Found" on the web.](https://en.wikipedia.org/wiki/HTTP_404)
+stands for "Not Found" on the web.](https://en.wikipedia.org/wiki/HTTP\_404)
```

Comment:
Is `_` really a reserved character here?

Reply:
Markdown has so many flavors and parsers that it's soon on the level of CSV. The package [remark-stringify](https://github.com/remarkjs/remark/tree/main/packages/remark-stringify) thinks this should be escaped as in some parsers it can be interpreted as beginning of emphasis. GitHub and Docsify understands it correctly, but this was updated by `npm run lint-fix` (via remark-stringify) and so I left it in.